### ISSUE TITLE
The >> should be removed from the link text 'Next 15 items'

### ISSUE THREAD

**Issue by ivanteoh** [eea/eea.facetednavigation#97]
Recently one of our 4.x plone sites was audited for WCAG 2.0 AA. The report came back and mentioned that the >> should be removed from the link text 'Next 15 items'. >> does not mean anything for the user that uses screen reader. Either remove it or replace/add the symbol using style sheet.

**Comment by avoinea** [comment]
The `>>` and `<<` come from Plone batching template, thus not related to eea.facetednavigation. Please address this issue to Plone itself.
